types/mithril/stream/index.d.ts

```diff
@@ -1,40 +1,49 @@
-		scan<T, U>(fn: (acc: U, value: T) => U, acc: U, stream: Stream<T>): Stream<U>;
-		/** Takes an array of pairs of streams and scan functions and merges all those streams using the given functions into a single stream. */
-		scanMerge<T, U>(pairs: Array<[Stream<T>, (acc: U, value: T) => U]>, acc: U): Stream<U>;
-		/** Takes an array of pairs of streams and scan functions and merges all those streams using the given functions into a single stream. */
-		scanMerge<U>(pairs: Array<[Stream<any>, (acc: U, value: any) => U]>, acc: U): Stream<U>;
-		/** A special value that can be returned to stream callbacks to halt execution of downstreams. */
-		readonly HALT: any;
-	}
+declare namespace Stream {
+	/** Creates a computed stream that reactively updates if any of its upstreams are updated. */
+	export function combine<T>(combiner: (...streams: any[]) => T, streams: Array<Stream<any>>): Stream<T>;
+	/** Combines the values of one or more streams into a single stream that is updated whenever one or more of the sources are updated */
+	export function lift<A, Z>(fn: (a: A) => Z, s: Stream<A>): Stream<Z>;
+	export function lift<A, B, Z>(fn: (a: A, b: B) => Z, sa: Stream<A>, sb: Stream<B>): Stream<Z>;
+	export function lift<A, B, C, Z>(fn: (a: A, b: B, c: C) => Z, sa: Stream<A>, sb: Stream<B>, sc: Stream<C>): Stream<Z>;
+	export function lift<A, B, C, D, Z>(fn: (a: A, b: B, c: C, d: D) => Z, sa: Stream<A>, sb: Stream<B>, sc: Stream<C>, sd: Stream<D>): Stream<Z>;
+	export function lift<A, B, C, D, E, Z>(fn: (a: A, b: B, c: C, d: D, e: E) => Z, sa: Stream<A>, sb: Stream<B>, sc: Stream<C>, sd: Stream<D>, se: Stream<E>): Stream<Z>;
+	export function lift<A, B, C, D, E, F, Z>(fn: (a: A, b: B, c: C, d: D, e: E, f: F) => Z, sa: Stream<A>, sb: Stream<B>, sc: Stream<C>, sd: Stream<D>, se: Stream<E>, sf: Stream<F>): Stream<Z>;
+	export function lift<T>(fn: (...values: any[]) => T, ...streams: Array<Stream<any>>): Stream<T>;
+	/** Creates a stream whose value is the array of values from an array of streams. */
+	export function merge(streams: Array<Stream<any>>): Stream<any[]>;
+	/** Creates a new stream with the results of calling the function on every incoming stream with and accumulator and the incoming value. */
+	export function scan<T, U>(fn: (acc: U, value: T) => U, acc: U, stream: Stream<T>): Stream<U>;
+	/** Takes an array of pairs of streams and scan functions and merges all those streams using the given functions into a single stream. */
+	export function scanMerge<T, U>(pairs: Array<[Stream<T>, (acc: U, value: T) => U]>, acc: U): Stream<U>;
+	/** Takes an array of pairs of streams and scan functions and merges all those streams using the given functions into a single stream. */
+	export function scanMerge<U>(pairs: Array<[Stream<any>, (acc: U, value: any) => U]>, acc: U): Stream<U>;
+	/** A special value that can be returned to stream callbacks to skip execution of downstreams. */
+	export const SKIP: any;
 }
 
-declare const Stream: Stream.Static;
```

Comment:
[![I like it](https://media.giphy.com/media/3o6ZtpvPW6fqxkE1xu/giphy.gif)](https://gph.is/2ar5hhh)